packages/blocks/src/api/raw-handling/figure-content-reducer.js

```diff
@@ -73,7 +73,7 @@ export default function( node, doc, schema ) {
 	let wrapper = nodeToInsert;
 
 	while ( wrapper && wrapper.nodeName !== 'P' ) {
-		wrapper = wrapper.parentElement;
+		wrapper = wrapper.parentNode;
```

Comment:
Hi @ellatrix :wave: Do you foresee any issues with this single line change on the web side of things? My concern is, as we traverse upwards, that this could alter the behavior on web, because it may traverse all the way to the document, rather than stopping at the body.

Reply:
Upon further inspection, I think it _should_ exhibit the same behavior, as even in the case where it traverses slightly higher, it ultimately leaves `wrapper` `undefined` or `null`, which then adds the `figure` to the original parent (i.e. the only case where `wrapper` is used, it is guaranteed to be a `<p>` element, where the `figure` is removed from it, and prepended to it). Can you confirm if my understanding is correct?

Reply:
This change seems ok to me, but it would be great if you could write a test case that fails in `master` and doesn't fail here.

Reply:
Thanks for taking a look! :) 

To clarify, do you mean a test case that fails on mobile? Maybe I should first clarify that this image omission only seems to occur on mobile (i.e. I tried pasting the same on web Gutenberg, and it successfully handles the image).

Reply:
Perhaps, yes. How do we prevent this from happening in the future?